driver/main.c

```diff
@@ -150,10 +160,16 @@ TRACEPOINT_PROBE(sched_switch_probe, bool preempt, struct task_struct *prev, str
 TRACEPOINT_PROBE(signal_deliver_probe, int sig, struct siginfo *info, struct k_sigaction *ka);
 #endif
 
-#ifdef CAPTURE_PAGE_FAULTS
+/* tracepoints `page_fault_user/kernel` don't exist on ARM64 architecture .*/
+#if defined(CAPTURE_PAGE_FAULTS) && !defined(CONFIG_ARM64)
```

Comment:
It seems that `CAPTURE_PAGE_FAULTS` is already off for arm: https://github.com/falcosecurity/libs/blob/c41b362c5c302ba75982ea0cd0b30d92933a3313/driver/ppm.h#L36